code/TemplateStudioForWinUICs/Templates/Ft/SettingsStorage/Param_ProjectName/Param_ProjectName_postaction.csproj

```diff
@@ -1,4 +1,4 @@
-﻿  </ItemGroup>
+  </ItemGroup>
```

Comment:
What's changed here?

Reply:
Encoding. Often when I touch files, the encoding gets broken.

Reply:
Is this a specific editor?
Can we do anything with `.editorconfig` to prevent this from happening?
Or could we run the encoding script as part of the CI to ensure that everything is always correctly encoded and we don't get these issues showing up in future PRs?

Reply:
> You don't ever hit this issue? 

Not, but I don't edit the template files in VS very often. It might be a difference in behavior with VS2022